pkg/constants/constants.go

```diff
@@ -35,30 +35,30 @@ const (
 	SidecarDefaultMemoryLimit = "128M"
 	PVCStorageSize            = "1Gi"
 
-	// WorkspaceIDLabel is label key to store workspace identifier
-	WorkspaceIDLabel = "controller.devfile.io/workspace_id"
+	// DevWorkspaceIDLabel is label key to store workspace identifier
+	DevWorkspaceIDLabel = "controller.devfile.io/devworkspace_id"
```

Comment:
I think `id` would be enough

Reply:
it's the time to make this label shorter if we wish, but I don't think we win anything with such change, just make things less clearer.
It would be clearer if it would be `devworkspace.devfile.io/name` but it's `controller.devfile.io/` group, and just name is not clear IMHO